src/components/Avatar.js

```diff
@@ -34,7 +37,11 @@ const Avatar = ({ collective, src, type = 'USER', radius, name, ...styleProps })
   if (collective) {
     type = collective.type;
     name = collective.name;
-    src = getCollectiveImage(collective);
+    if (collective.isIncognito) {
+      src = defaultImage['ANONYMOUS'];
+    } else if (collective.image) {
+      src = `${getBaseImagesUrl()}/${collective.slug}/avatar.png`;
```

Comment:
Why remove `getCollectiveImage`, mistake when merging?

Reply:
Possible. There's been so many rebase / conflicts / ... over the course of a month... 

the `getCollectiveImage` wasn't handling the default anonymous avatar.
Also it wasn't clear the purpose of this function (it's not documented) when I looked at it.
Do you me to replace that line 43 with `src = getCollectiveImage(collective);` ?

Reply:
Yes